tinkerhost.net.ggssl-verification.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "tinkerhost.net",
+  "providerName": "TinkerHost",
+  "serviceId": "ggssl-verification",
+  "serviceName": "TinkerHost",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.tinkerhost.net",
+  "syncRedirectDomain": "app.tinkerhost.net",
+  "logoUrl": "https://app.tinkerhost.net/assets/image/th.svg",
+  "description": "Verifies a domain name with our SSL provider, GoGetSSL",
+  "variableDescription": "host describes the subdomain at which the verification key is avaiable. destination describes the second part of the verification key",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "_%host%",
```

Comment:
I don't think this usage of special variable %host% is intended here.

Please take a look at the output from the [online editor](https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit)

<img width="1224" alt="image" src="https://github.com/user-attachments/assets/52a7e1ff-f862-4fd4-9846-0478157ae1b4">

Reply:
Fixed in [3500533](https://github.com/Domain-Connect/Templates/pull/566/commits/35005332f5fe7e001aeeb94e99e54a0e1ee38a45)